src/main/java/az/ingress/mscategory/exception/ErrorHandler.java

```diff
@@ -0,0 +1,36 @@
+package az.ingress.mscategory.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import static org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+
+@Slf4j
+@RestControllerAdvice
+public class ErrorHandler {
+
+    @ExceptionHandler(Exception.class)
+    @ResponseStatus(INTERNAL_SERVER_ERROR)
+    public ErrorResponse errorHandler(Exception exception) {
+        log.error("ActionLog.errorHandler.error", exception);
```

Comment:
1.rename to handle 2.  in log write exception class name and stacktrace 3. use static import for enums